Description: The X.org X server 1.17 doesn't listen on TCP by default, we need to pass -listen to enable it
Author: Robert Ancell <robert.ancell@canonical.com>
Bug: https://bugs.launchpad.net/bugs/1449282

=== modified file 'src/x-server-local.c'
--- a/src/x-server-local.c	2015-05-26 02:11:04 +0000
+++ b/src/x-server-local.c	2015-10-12 13:41:49 +0000
@@ -490,6 +490,7 @@
     /* Connect to a remote server using XDMCP */
     if (server->priv->xdmcp_server != NULL)
     {
+        g_string_append (command, " -listen tcp");
         if (server->priv->xdmcp_port != 0)
             g_string_append_printf (command, " -port %d", server->priv->xdmcp_port);
         g_string_append_printf (command, " -query %s", server->priv->xdmcp_server);
@@ -498,6 +499,8 @@
     }
     else if (!server->priv->allow_tcp)
         g_string_append (command, " -nolisten tcp");
+    else
+        g_string_append (command, " -listen tcp");
 
     if (server->priv->vt >= 0)
         g_string_append_printf (command, " vt%d -novtswitch", server->priv->vt);

